### DISCUSSION TITLE
Persistant Workspaces After Update

### DISCUSSION THREAD

**Discussion by jhenrique04** [hyprwm/Hyprland#3789] [Q&A] [answered]
Hello, so, I'm using Hyprland for a while and I'm also using Waybar.

Before yearstuday, Waybar would only show the workspaces that I was working with (workspaces that had something open), but then I updated my system using paru -Syu, and now i have 10 workspaces showing in Waybar, even if I'm not using them.

![image](https://github.com/hyprwm/Hyprland/assets/115225048/6bf56e53-c0bc-4b46-8213-a118279568bd)

Already did a little research trying to find a solution, but didn't find a fix for it, tbh I don't know if the problem is in Hyprland or Waybar, if someone knows something that could help, I'ld be glad if shared.

Thanks in advice!

**Comment by nabajour** [comment, marked as answer]
I noticed the same, waybar got a new release that updated the package on Debian unstable last week.

I had the key `"persistent-workspace" `set in my waybar config for `"hyprland/workspaces"`. Unsetting it and setting `"active-only": false` and `"all-output": false` fixed it here. Check the config in [the waybar wiki](https://github.com/Alexays/Waybar/wiki/Module:-Hyprland).``

**Reply by jhenrique04** [reply]
Nice! worked for me too, thank you so much.
